packages/features/ee/common/components/LicenseRequired.tsx

```diff
@@ -46,6 +46,7 @@ const LicenseRequired = ({ children, as = "", ...rest }: LicenseRequiredProps) =
                     consoleUrl: `<a href="https://go.cal.com/console" target="_blank" class="underline">
                 ${APP_NAME}
               </a>`,
+                    setupUrl: `<a href="${WEBAPP_URL}/auth/setup">/auth/setup</a>`,
```

Comment:
Text now refers to the admin wizard to be able to enter a license key instead of the environment variable.